src/core/server/elasticsearch/client/get_ecs_response_log.test.ts

```diff
@@ -68,6 +68,27 @@ describe('getEcsResponseLog', () => {
       `);
     });
 
+    test('redacts x-elastic-app-auth headers by default', () => {
+      const event = createResponseEvent({
+        requestParams: { headers: { 'x-elastic-app-auth': 'hello', 'user-agent': 'world' } },
+        response: { headers: { 'content-length': '123' } },
```

Comment:
I guess the filter works for `response` as well, correct? Should we test it?

Reply:
This header should only ever appear in the request, but I've updated the test to include response headers too -- for the sake of completeness. But yeah, the filters apply to _all_ headers, request or response.